src/components/MainPage/mainPage.js

```diff
@@ -10,10 +10,10 @@ const boggle = require('pf-boggle');
 export default function MainPage() {
     const boggleArray = boggle.generate(4, boggle.diceSets['classic4']);
     const allAnswer = boggle.solve(boggleArray);
-    const boggleAnswer = allAnswer.filter(words => words.word.length > 2);
+    const boggleAnswer = allAnswer.filter((words) => words.word.length > 2);
 
     useEffect(() => {
-        console.log("actualAnswer: ", boggleAnswer)
+        console.log('actualAnswer: ', boggleAnswer);
```

Comment:
at some point we prolly wanna remove all the console statements

Reply:
Yeah, we can leave it for now since we're still in the trial and error phase